Frontend/src/components/TaskList.tsx

```diff
@@ -23,29 +24,21 @@ export default function TaskList() {
       <div>
         <h1>Pending</h1>
         <ul>
-          {tasks
-            .filter((task) => task.isCompleted === false)
-            .map((task) => {
-              return (
-                <li key={task.id}>
-                  {task.id}) {task.title}
-                </li>
-              );
-            })}
+          {tasks.map((task) =>
```

Comment:
I think there is still some duplication between #27-31 and #37-41, but it's much better now anyway!